packages/mobile/types/exercises.ts

```diff
@@ -0,0 +1,8 @@
+export interface Exercise {
+  id: number;
+  name: string;
+}
+
+export interface ExercisesProps {
```

Comment:
is there benefit to having these here and not on the component itself?

Reply:
Probably don't need ExercisesProps now that I'm thinking about it. Can remove that and just keep Exercise